crates/core/src/registry.rs

```diff
@@ -198,18 +199,20 @@ pub async fn get_all_versions_descending(dependency_name: &str) -> Result<Versio
 
     match revision
         .data
-        .iter()
+        .par_iter()
         .map(|r| Version::parse(&r.version))
         .collect::<std::result::Result<Vec<Version>, _>>()
     {
         Ok(mut versions) => {
             // all versions are semver compliant
-            versions.sort_unstable_by(|a, b| b.cmp(a)); // sort in descending order
+            versions.par_sort_unstable_by(|a, b| b.cmp(a)); // sort in descending order
```

Comment:
I quickly benchmarked and there is no benefit to sorting in parallel here:

![image](https://github.com/user-attachments/assets/4e3cb0bf-45af-44d1-9fc2-bad90b3182eb)